packages/ipfs-core/src/components/files/utils/add-link.js

```diff
@@ -8,6 +8,7 @@ const CID = require('cids')
 const log = require('debug')('ipfs:mfs:core:utils:add-link')
 const UnixFS = require('ipfs-unixfs')
 const DirSharded = require('ipfs-unixfs-importer/src/dir-sharded')
+const defaultImporterOptions = require('ipfs-unixfs-importer/src/options')
```

Comment:
shouldn't this be exported by the entrypoint to avoid these deep requires ?

Reply:
That's happening here: https://github.com/ipfs/js-ipfs-unixfs/pull/122

Reply:
This has been done